lib/network/modules/InteractionHandler.js

```diff
@@ -504,15 +504,7 @@ class InteractionHandler {
   onMouseWheel(event) {
     if (this.options.zoomView === true) {
       // retrieve delta
-      let delta = 0;
-      if (event.wheelDelta) { /* IE/Opera. */
-        delta = event.wheelDelta / 120;
-      }
-      else if (event.detail) { /* Mozilla case. */
-        // In Mozilla, sign of delta is different than in IE.
-        // Also, delta is multiple of 3.
-        delta = -event.detail / 3;
-      }
+      let delta = event.deltaY;
```

Comment:
The keyword let screams that this variable will be reassigned but it never is. Always use const unless you reassign the variable later.